src/azul/security/auth.py

```diff
@@ -0,0 +1,102 @@
+# See https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/Security/Authentication%20and%20Authorization/Setting%20up%20DCP%20Auth
+from logging import getLogger
+from typing import Dict, Any
+from urllib.parse import urlencode
+
+from azul import config
+from azul.security.verifier import (verify,
+                                    InvalidTokenError,
+                                    NonDecodableTokenError)
+
+logger = getLogger(__name__)
+
+fusillade_callback_url = f'{config.service_endpoint()}/auth/callback'
```

Comment:
```suggestion
fusillade_callback_url = {config.service_endpoint() + '/auth/callback'
```

Reply:
Reads more clearly due to the whitespace around +

Reply:
Oh... I see. This will be changed in the next commit.